### ISSUE TITLE
Fix errp / errn mixup

### ISSUE THREAD

**Issue by cdeil** [gammapy/gamma-cat#109]
@GernotMaier - I see that in `input/schemas/dataset_source_info.schema.yaml` you introduced this:
```
      period:
        type: object
        additionalProperties: false
        description: Orbital period
        properties:
          val: {type: [number, string], description: Measured value}
          err: {type: [number, string], description: Statistical error}
          errp: {type: [number, string], description: Lower statistical error}
          errn: {type: [number, string], description: Upper statistical error}
          err_sys: {type: [number, string], description: Systematic error}
```
This is incorrect, we always use "p" for "positive" or "upper" statistical error, and "n" for "negative" or "lower" statistical error.

I could correct this in the schema, but I'm not sure which error is which in the files where you did data entry. Could you please check and put the correct numbers in these files?
```
$ grep errp input/data/*/*/*.yaml
input/data/2014/2014ApJ...780..168A/tev-000030-1.yaml:#    period: {val: 315, errp: 4, errn: 6 }
input/data/2014/2014ApJ...780..168A/tev-000030-2.yaml:    period: {val: 315, errp: 4, errn: 6 }
input/data/2014/2014ApJ...780..168A/tev-000030-3.yaml:    period: {val: 315, errp: 4, errn: 6 }
input/data/2014/2014ApJ...780..168A/tev-000030-4.yaml:    period: {val: 315, errp: 4, errn: 6 }
input/data/2014/2014ApJ...780..168A/tev-000030-5.yaml:    period: {val: 315, errp: 4, errn: 6 }
input/data/2014/2014ApJ...780..168A/tev-000030-6.yaml:    period: {val: 315, errp: 4, errn: 6 }
input/data/2016/2016arXiv161003751S/tev-000030-1.yaml:    period: {val: 315, errp: 4, errn: 6 }
input/data/2016/2016arXiv161003751S/tev-000030-2.yaml:    period: {val: 315, errp: 4, errn: 6 }
input/data/2016/2016arXiv161003751S/tev-000030-3.yaml:    period: {val: 315, errp: 4, errn: 6 }
```

**Comment by GernotMaier** [comment]
Yes - mixed this up. Fixed the upper/lower values in my fork.

**Comment by cdeil** [comment]
@GernotMaier - I see that you've changed the data files in 8005bee52599d3efaf3651dcee2e9f1105b28a90 .
Just now, I fixed the schema file in 8bcb4aa .

All is good now as far as I can see. Closing this issue now.
